### ISSUE TITLE
Drachenöde - Auf rubinroten Schwingen

### ISSUE THREAD

**Issue by Ysanne313** [elixyr-wow/Bugtracker#32]
Bei der Quest Auf rubinroten Schwingen [https://de.wowhead.com/quest=12498/auf-rubinroten-schwingen](url) tötet man den NPC [https://de.wowhead.com/npc=28018/thiassi-der-blitzbringer](url). danach sollte der NPC Großnekrolord Antiok [https://de.wowhead.com/npc=28006/gro%C3%9Fnekrolord-antiok](url) von dem Thiassi runterspringen und angreifbar sein,  er despawnt mit dem bleben Thiassis und somit kann die quest nicht abgeschlossen werden

**Comment by Ysanne313** [comment]
--Update -- der Nekrolord Antiok ist angreifbar auf dem Rücken von Thiassi und  die Sense kann nahc seinem Tod Gelootet werden.

**Comment by DarkBrain2580** [comment]
Ok also closen?

**Comment by Ysanne313** [comment]
nein, is zu nervig / schwer wenn mans nit genau weiß, der riese muss man mim drachen umhaun und dann komtm der untote runter, und im mom wie es is  is ne grp zu empfeheln
